src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py

```diff
@@ -201,7 +202,7 @@ def store_schema_properties(dynamodb: DynamoDBServiceResource, schema: Schema) -
 
 def provision_schemas(master_graph_client, raw_schemas):
```

Comment:
Should we add a None/Any return type here too?

Reply:
This is all copy pasted from the Provision service - gross - I decided to not waste too much time on this part